cachet-tools/status-updater-template.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: template.openshift.io/v1
+kind: Template 
+labels:
+  template: hosted-services-status-updater-template # Should it be called insights-platform-tools-cachet
+message: A new cachet-updter application has been created in your project.
+metadata:
```

Comment:
I think you can remove all the metadata except `name`.

Reply:
same as above.  removed the file.